packages/ast/README.md

```diff
@@ -51,9 +51,8 @@ console.log(signatures);
 - function `ifInstruction(testLabel, result, test, consequent, alternate)`
 - function `withLoc(n, end, start)`
 - function `moduleImport(module, name, descr)`
-- function `globalImportDescr(valtype, mutability)`
+- function `globalType(valtype, mutability)`
```

Comment:
This is auto generated by the `dump-export` package. Basically it lists every export.

Reply:
Ah - thanks for the heads-up. Eventually I'd like to generate this from flow documentation, giving a more rich API doc